### PR TITLE
Added basic gradle setup for screenshot-tests-for-android

### PR DESCRIPTION
**Pull request by grubdragon**
#1826 Tried to add the gradle setup, hopefully didn't mess up. Would love a review. @mcomella @Sdaswani

### CODE REVIEW

**Comment by Sdaswani** [comment]
@npark-mozilla is this something you can review while @pocmo is on pat leave?

**Comment by npark-mozilla** [comment]
@Sdaswani : @pocmo would know better regarding the gradle stuff, but this seems pretty harmless, I can take a look.

File: app/build.gradle [hunk 1]
```diff
@@ -183,6 +184,10 @@ dependencies {
     debugCompile "com.squareup.leakcanary:leakcanary-android:$leakcanary_version"
     compile "com.squareup.leakcanary:leakcanary-android-no-op:$leakcanary_version"
 
+    // screenshot-tests-for-android : Layout Hierarchy Plugin dependencies
```
Comment (npark-mozilla) [inline app/build.gradle]: If this is used for UI Tests, then perhaps we could use androidTestCompile instead of compile command?

Comment (grubdragon) [reply app/build.gradle]: Seems like http://facebook.github.io/screenshot-tests-for-android/ this page compiles normally. But I have very little idea about gradle, so I'm willing to change it, if need be.

Comment (npark-mozilla) [reply app/build.gradle]: it is a convention for libs that are used for UI Tests - it's more for the style of the gradle script.  If it works, it'd be great if we change it.

**Comment by Sdaswani** [comment]
@grubdragon i think @npark-mozilla had a suggestion here? https://github.com/mozilla-mobile/focus-android/pull/2060#discussion_r162790267

**Comment by pocmo** [comment]
Thank you for this PR @grubdragon. Do you have experience in running this tool? Specifically in automation. This PR looks like a good first step - but without actually running the tool (and reporting the results) I'm not sure if we gain much by merging this as-is.

**Pull request closed without merging**